planner/core/explain.go

```diff
@@ -322,7 +322,13 @@ func (p *PhysicalTableScan) isFullScan() bool {
 		return false
 	}
 	for _, ran := range p.Ranges {
-		if !ran.IsFullRange() {
+		var unsignedIntHandle bool
+		if p.Table.PKIsHandle {
```

Comment:
This can be treated as a tiny bug-fix?

Reply:
Yes, so do we need to split it to another pr?

Reply:
Here is okay.